src/Glpi/Controller/Knowbase/KnowbaseItemController.php

```diff
@@ -95,7 +95,7 @@ public function full(Request $request): Response
     }
 
     #[Route(
-        "/Knowbase/KnowbaseItem/SearchSolution/{itemtype}/{items_id}",
+        "/Knowbase/KnowbaseItem/Search/{itilobject}/{itemtype}/{items_id}",
         name: "knowbaseitem_search_solution",
```

Comment:
Check that there are no others left:
```suggestion
        name: "knowbaseitem_search_itilobject",
```